### PR TITLE
chore: Upgrade dredd and increase hooks timeout

### LINKED ISSUE #6372: Increase dredd timeout
dredd has been timing out a lot lately. Increase the timeouts

Also, dredd 4.0 is ancient. It hasn't been updated since dredd tests were added in the repo. Use latest one

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #6372

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Did you just increase the version without even verifying that it works?

**Comment by kushthedude** [comment]
> Did you just increase the version without even verifying that it works?

I was checking if it would directly, ALAS I was wrong, There a lot of depreciation and some removals too, I will fix on local

**Comment by iamareebjamal** [comment]
We don't want silent mode.

**Comment by kushthedude** [comment]
@iamareebjamal Its ready !

File: dredd.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 dry-run: null
 hookfiles: ./tests/hook_main.py
 language: python
-sandbox: false
+# sandbox: false
```
Comment (iamareebjamal) [inline dredd.yml]: Remove
```suggestion
```

**Comment by kushthedude** [comment]
> We don't want silent mode.

Info mode has been removed, So I went with the `warning` as `debug` was extending the log far Travis limits

File: .travis.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ before-install:
 
 install:
   - npm install -g aglio
-  - npm install -g dredd@4.0.0
+  - npm install -g dredd@11.0.0
```
Comment (iamareebjamal) [inline .travis.yml]: This is still old. Use latest

```suggestion
  - npm install -g dredd
```

Comment (kushthedude) [reply .travis.yml]: Its LTS, I used the stable one .

Comment (kushthedude) [reply .travis.yml]: Done !

**Pull request merged**